test/testDominatingSet.cpp

```diff
@@ -0,0 +1,185 @@
+// #include <iostream>
+#include <dominatingset/RooijBodlaenderMDS.hpp>
+#include <dominatingset/BranchAndReduceMDS.hpp>
+#include <dominatingset/BranchAndReduceMSC.hpp>
+#include <dominatingset/SchiermeyerMDS.hpp>
+#include <dominatingset/FominKratschWoeginger.hpp>
+#include <dominatingset/ExhaustiveMDS.hpp>
+#include <chrono>
+#include <io/G6GraphReader.hpp>
+#include <fstream>
+#include<algorithm>
+#include<iterator>
```

Comment:
Once it's #include <A>, another time #include<A>, please unify it with reference to the previous files (e.g. take recognition/Perfect files as a reference)